WebDriverAgentLib/Utilities/FBXPath.m

```diff
@@ -224,6 +224,13 @@ + (int)recordElementAttributes:(xmlTextWriterPtr)writer forElement:(XCElementSna
       return rc;
     }
   }
+  if (element.wdClass) {
```

Comment:
this is going to break several unit and integration tests. Please check and update these